realm/realm-library/build.gradle

```diff
@@ -620,6 +684,13 @@ task bintrayUpload() {
     group = 'Publishing'
 }
 
+task ojoUpload() {
```

Comment:
Only release variants as far as I am aware, since this is a proper release just with the "SNAPSHOT" modifier.